sim.html

```diff
@@ -68,8 +68,16 @@
 			</table>
 		</div>
 		<div class="controlpanel">
-			<input type="checkbox" id="online" name="online" checked autocomplete="off" />
-			<label for="online">Online/offline</label>
+			<p>Configuration:</p>
+			<div id="online">
+				<input type="checkbox" name="online" checked autocomplete="off" />
+				<label for="online">Online/offline</label>
+			</div>
+			<div id="experiment-trending">
+				<input type="checkbox" name="experiment-trending" />
```

Comment:
hmm, i should make a record whether on/off the next time user visit the simulator. working on it.